### PR TITLE
Filesystem fix

### PR DESCRIPTION
**Pull request by jwangdx**
### What I have done

Fixed test error in file system `dirMetadataUpdate`. Update comments.

### CODE REVIEW

**Comment by jwangdx** [comment]
Added one test case and removed unnecessary disrupter codes.

File: storage/storageclient/filesystem/api_test.go [hunk 1]
```diff
@@ -5,15 +5,20 @@
...
-	"github.com/DxChainNetwork/godx/storage/storageclient/filesystem/dxdir"
 	"math"
+	"path/filepath"
+	"reflect"
 	"strconv"
 	"strings"
 	"testing"
 	"time"
+
+	"github.com/davecgh/go-spew/spew"
```
Comment (mzhangdx) [inline storage/storageclient/filesystem/api_test.go]: import format.

based on `goimports` tool, github.com/davecgh/go-spew/spew should be placed at the end

File: storage/storageclient/filesystem/generate.go [hunk 2]
```diff
@@ -9,6 +9,7 @@ import (
 	"math/rand"
 	"os"
 	"path/filepath"
+	"runtime"
```
Comment (mzhangdx) [inline storage/storageclient/filesystem/generate.go]: double check the import order

File: storage/storageclient/filesystem/api_test.go (hunk 1 above)
Comment (jwangdx) [reply storage/storageclient/filesystem/api_test.go]: The import sequence are supposed to be following:

1. Original package by golang
2. Third party package
3. Project package.

Here the file import strictly follows the order and I cannot see the point. Please specify. Thx

File: storage/storageclient/filesystem/generate.go (hunk 2 above)
Comment (jwangdx) [reply storage/storageclient/filesystem/generate.go]: Formatted. Thx.

File: storage/storageclient/filesystem/api_test.go (hunk 1 above)
Comment (mzhangdx) [reply storage/storageclient/filesystem/api_test.go]: According to the official golang [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#imports)

`goimports` should be used to ensure the correct order of the imported packages

Comment (jwangdx) [reply storage/storageclient/filesystem/api_test.go]: Agreed and fixed

**Pull request merged**